scripts/beachball/base.config.json

```diff
@@ -10,6 +10,7 @@
     "**/__fixtures__/**",
     "**/__mocks__/**",
     "**/common/isConformant.ts",
```

Comment:
@TristanWatanabe  this should be removed  once all packages have been migrated or is this still used in v8 ?

`"**/SPEC*.md",` should be tweaked as well

Reply:
common/isConformant.ts is still used by v8